scss/core/_variables.scss

```diff
@@ -873,3 +908,37 @@ $list-group-action-hover-color:     $list-group-action-color !default;
 
 $list-group-action-active-color:    $body-color !default;
 $list-group-action-active-bg:       $gray-200 !default;
+
+:root {
+  @each $color, $value in $colors {
+    --pgn-color-#{$color}: #{$value};
+  }
+
+  @each $color, $value in $theme-colors {
+    --pgn-color-#{$color}: #{$value};
+  }
+
+  @each $color, $value in $theme-color-levels {
+    --pgn-color-#{$color}: #{$value};
+  }
+
+  @each $color, $value in $element-color-levels {
+    --pgn-color-#{$color}: #{$value};
+  }
+
+  @each $size, $length in $spacers {
+    --png-spacer-#{$size}: #{$length};
```

Comment:
```suggestion
    --pgn-spacer-#{$size}: #{$length};
```

Reply:
This is no longer needed with the most recent changes.